controllers/upload.go

```diff
@@ -25,14 +25,8 @@ func GeneratePresignedURL(c *gin.Context) {
 	// Generate a New Video ID
 	videoID := utils.VideoIDGen(extension)
 
-	// Create a new session
-	sess, _ := session.NewSession(&aws.Config{
-		Region: aws.String(constants.S3_REGION),
-	})
-	svc := s3.New(sess)
-
```

Comment:
Here a new s3 session was being created for every new API call, So i refactored it to create a session only once when the router is being initiated and pass on that same session to the controllers. Through this the same session will be used throughout

Reply:
Cool